docs/INSTALL.md

```diff
@@ -93,6 +93,45 @@ sudo update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.gold" 20
 sudo update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.bfd" 10
 ```
 
+### Dependencies: Peppermint 7 / Ubuntu 18.x - i386 / i686 using LDC (Optional)
+**Important:** Build environment must have at least 512 of memory & 1GB swap space
+
+**Important:** Only use this method if you have <1GB of physical memory.
+
+**Note:** Peppermint 7 validated with the LDC compiler on the following i386 / i686 platform:
```

Comment:
Not sure why this should only be used with low memory? LDC(2) is quite good, produces smaller binaries, and I haven't seen any misbehaviour ever since.

Reply:
Happy for you to tweak the document as you see fit :) The aim here was just to be consistent :)